src/cursor.rs

```diff
@@ -1,5 +1,6 @@
 use libc::{c_void, size_t, c_uint};
-use std::{ptr, slice};
+use std;
```

Comment:
The import of `std` shouldn't be necessary if you change `std::fmt::Debug` to `fmt::Debug` below.